src/UI/Art/Animation/NobunagaBattleAnimation.java

```diff
@@ -30,6 +37,11 @@ public boolean animate(Graphics g)
   {
     long animTime = System.currentTimeMillis() - startTime;
 
+    // Draw the attacker in position.
+    int spriteIndex = 0; // No need to be fancy.
+    boolean flip = attackerX > defenderX;
+    actorSpriteSet.drawUnit(g, UnitSpriteSet.AnimState.IDLE, spriteIndex, attackerX * tileSize, attackerY * tileSize, flip );
```

Comment:
So the attacker always faces east, except when east of the defender.
But the defender doesn't turn to face the attacker in any case.
I think I'd prefer just using the CO's facing as we did in the past.

Reply:
Yeah, I agree. I didn't have that here though, since `SpriteMapView` owns it. Probably fixable if we like the direction of this change otherwise.